src/bokeh/io/export.py

```diff
@@ -318,7 +318,15 @@ def get_layout_html(obj: UIElement | Document, *, resources: Resources = INLINE,
     """
 
     def html() -> str:
-        return file_html(obj, resources, title="", template=template, theme=theme, suppress_callback_warning=True, _always_new=True)
+        return file_html(
+            obj,
+            resources=resources,
+            title="",
+            template=template,
+            theme=theme,
+            suppress_callback_warning=True,
+            _always_new=True,
```

Comment:
I've forgotten what this was about, I wonder if it is something that can be dispensed with at some point

Reply:
It's only used in `bokeh.io.export.get_layout_html()`, but there's no explanation in the code why.